src/simplewallet/simplewallet.cpp

```diff
@@ -4719,7 +4719,7 @@ bool simple_wallet::stake_all(const std::vector<std::string> &args_)
   size_t mixins = DEFAULT_MIX;
 
   uint64_t unlock_block = 0;
-  uint64_t locked_blocks = STAKING_REQUIREMENT_LOCK_BLOCKS;
+  uint64_t locked_blocks = STAKING_REQUIREMENT_LOCK_BLOCKS + 20; /* DO NOT COMMIT: Because the time you submit and the time the block gets accepted is always going to be > 1, which fails the is_reg_tx check */
```

Comment:
Why do not commit? This buffer needs to be included in cryptonote_config.h:

     52 #define STAKING_REQUIREMENT_LOCK_BLOCKS                 (30*24*31)                                                                                                                                          | 723         tie(mixRing, index) = populateFromBlockchain(inPk, mixin);
     53 #define STAKING_REQUIREMENT_LOCK_BLOCKS_EXCESS          20                                                                                                                                                  | 724         return genRct(message, inSk, destinations, amounts, mixRing, amount_keys, kLRki, msout, index, outSk, false, hwdev);
     54 #define STAKING_RELOCK_WINDOW_BLOCKS                    (30*6)

Reply:
Updated.